diff --git a/lib/templates/formula/go-package.ts b/lib/templates/formula/go-package.ts
index b3bb680..98c0bb8 100644
--- a/lib/templates/formula/go-package.ts
+++ b/lib/templates/formula/go-package.ts
@@ -9,7 +9,7 @@ ${p.licenseLine}${p.urlLines}  head "https://github.com/${p.fullName}.git", bran
 ${p.livecheckBlock}${p.allbrewDependency ? `  depends_on "${p.allbrewDependency}"\n` : ""}  depends_on "go" => :build
 
   def install
-    system "go", "build", *std_go_args(ldflags: "-s -w")
+    system "go", "build", *std_go_args(ldflags: "-s -w")${p.goBuildExtraArgs || ""}
   end
 
 ${p.serviceBlock}  test do
